### ISSUE TITLE
watch is not working if override ionic_copy

### ISSUE THREAD

**Issue by ErnestRoca** [driftyco/ionic-app-scripts#581]
#### Short description of the problem:
When I override the copy task in package.json, the watch command stops working

Only works overriding Cmd-line Flag like this:

```
 "scripts": {
     "ionic:build": "ionic-app-scripts build --copy ./config/copy.config.js"
  }
```
#### What behavior are you expecting?

watch task working when I change any ts, html or scss file.

**Steps to reproduce:**
1. Add this in your package.json:

 ``` 
"config": {
    "ionic_copy": "./config/copy.config.js"
   }
```
2. Change any ts, html or scss file in your project
3. watch task is not executed

```
insert any relevant code between the above and below backticks
```

**Which @ionic/app-scripts version are you using?**

0.0.47

**Comment by danbucholtz** [comment]
Hi @ErnestRoca,

This is resolved in `0.0.48` which was just published.

Please let me know if you have any issues.

Thanks,
Dan

**Comment by ErnestRoca** [comment]
Thanks @danbucholtz !

This is working again:
 ``` 
"config": {
    "ionic_copy": "./config/copy.config.js"
   }
```
